images/openshift-kubernetes-nmstate-operator.yml

```diff
@@ -13,6 +13,7 @@ content:
 distgit:
   component: ose-kubernetes-nmstate-operator-container
   bundle_component: ose-kubernetes-nmstate-operator-bundle-container
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
 enabled_repos:
```

Comment:
Should repos be moved to rhel 9 as well?

Reply:
yep